js/src/personal/riyuan/Riyuan.module.css

```diff
@@ -13,7 +13,7 @@
 
 .appshell{
   margin-left: 88px;
-  margin-right: 88px;
+  margin-right: 2vh;
```

Comment:
Generally shouldn't use vh for things that are that small

Reply:
I try to use the pixel, but when i try to make the view port smaller, it stay at 88px, and it look weird with smaller screen